packages/mco/components/modals/app-manage-policies/helper/pvc-details-wizard-content.tsx

```diff
@@ -73,13 +74,14 @@ const getLabelsFromSearchResult = (searchResult: SearchResult): string[] => {
 
 const getLabelsFromTags = (tags: TagsType, currIndex: number): string[] =>
   tags.reduce((acc, tag, index) => {
-    const labels: string[] = (tag?.[1] || []) as string[];
+    const labels: string[] = (tag?.[NameValueEditorPair.Value] ||
+      []) as string[];
     return currIndex !== index ? [...acc, ...labels] : acc;
   }, []) as string[];
 
 const getPlacementsFromTags = (tags: TagsType, currIndex: number): string[] =>
   tags.reduce((acc, tag, index) => {
-    const placement: string = tag?.[0] as string;
+    const placement: string = tag?.[NameValueEditorPair.Name] as string;
```

Comment:
nits: already doing `const placement: string =`...
```suggestion
    const placement: string = tag?.[NameValueEditorPair.Name];
```